src/test/fixtures/profiles/gecko-profile.js

```diff
@@ -106,10 +106,22 @@ const thread = {
   markers: {
     schema: { name: 0, time: 1, data: 2 },
     data: [
-      [4, 0, { category: 'VsyncTimestamp', vsync: 0 }],
       [
-        5,
-        2,
+        // Ending a tracing but never starting it.
+        // Please keep it at the start if you add more markers in this structure
```

Comment:
This comment might be missed, perhaps add it to line 109 instead, so that it's where someone would add a marker?